index.bs

```diff
@@ -118,43 +95,28 @@ The Gyroscope Interface {#gyroscope-interface}
 <pre class="idl">
   [Constructor(optional SensorOptions sensorOptions)]
   interface Gyroscope : Sensor {
-    readonly attribute GyroscopeReading? reading;
+    readonly attribute unrestricted double? x;
+    readonly attribute unrestricted double? y;
+    readonly attribute unrestricted double? z;
   };
 </pre>
 
 To <dfn>Construct an Gyroscope Object</dfn> the user agent must invoke the 
 <a>construct a Sensor object</a> abstract operation.
 
-The GyroscopeReading Interface {#gyroscope-reading-interface}
----------------------------------------
-
-<pre class="idl">
-  [Constructor(GyroscopeReadingInit GyroscopeReadingInit)]
-  interface GyroscopeReading : SensorReading {
-      readonly attribute unrestricted double x;
-      readonly attribute unrestricted double y;
-      readonly attribute unrestricted double z;
-  };
-  
-  dictionary GyroscopeReadingInit {
-      unrestricted double x = 0;
-      unrestricted double y = 0;
-      unrestricted double z = 0;
-  };
-</pre>
-
-### The GyroscopeReading constructor ### {#gyroscope-reading-constructor}
-
-### The attribute ### {#gyroscope-reading-attribute}
+### The attribute ### {#gyroscope-attribute}
```

Comment:
I'd title that differently. See how it's generally done in other specs: one title per attribute.